elkserver/logstash/conf.d/20-filter-redir-haproxy_logstash.conf

```diff
@@ -101,88 +104,103 @@ filter {
     }
 
     # map header values onto dedicated fields and split the values of the headersall field into an array
-    if [redirtraffic.headersall] {
+    if [http][headers][all] {
       # map to dedicated fields
       grok {
-        match => { "redirtraffic.headersall" => [ "(?<redirtraffic.headeruseragent>([^|]*))\|(?<redirtraffic.headerhost>([^|]*))\|(?<redirtraffic.headerxforwardedfor>([^|]*))\|(?<redirtraffic.headerxforwardedproto>([^|]*))\|(?<redirtraffic.headerxhost>([^|]*))\|(?<redirtraffic.headerforwarded>([^|]*))\|(?<redirtraffic.headervia>([^|]*))" ] }
+        match => { "[http][headers][all]" => [ "(?<[http][headers][useragent]>([^|]*))\|(?<[http][headers][host]>([^|]*))\|(?<[http][headers][x_forwarded_for]>([^|]*))\|(?<[http][headers][x_forwarded_proto]>([^|]*))\|(?<[http][headers][x_host]>([^|]*))\|(?<[http][headers][forwarded]>([^|]*))\|(?<[http][headers][via]>([^|]*))" ] }
       }
-      
+
       # split the values into an array
       mutate {
-        split => { "redirtraffic.headersall" => "|" }
+        split => { "[http][headers][all]" => "|" }
       }
     }
-    
+
     # Set the timestamp from the log to @timestamp, example: 15/Apr/2018:19:22:31 +0000
     date {
-      match => [ "redirtraffic.timestamp", "dd/MMM/yyyy:HH:mm:ss Z" ]
+      match => [ "[redir][timestamp]", "dd/MMM/yyyy:HH:mm:ss Z" ]
       target => "@timestamp"
       timezone => "Etc/UTC"
     }
 
     # When IPv6 is enabled on your HAProxy host, IPV4 addresses are reported like ::ffff:ipv4address. Here we cut off the ::ffff: part
-    if "ffff" in [redirtraffic.sourceip] {
-      mutate {
-        gsub => [
-          "redirtraffic.sourceip", "\:\:ffff\:", ""
-        ]
-      }
-    }
-    if "ffff" in [destination.ip] {
+    if "ffff" in [source][ip] {
       mutate {
         gsub => [
-          "destination.ip", "\:\:ffff\:", ""
+          "[source][ip]", "\:\:ffff\:", ""
         ]
       }
     }
-    if "ffff" in [redirtraffic.sourceipcdn] {
+    # Apparently never extracted
+    # if "ffff" in [destination][ip] {
+    #   mutate {
+    #     gsub => [
+    #       "[destination][ip]", "\:\:ffff\:", ""
+    #     ]
+    #   }
+    # }
```

Comment:
destination.ip field doesn't seem to be extracted anywhere. Commented out and proposition to remove it if not needed.

Reply:
Could be a simple error, or an artefact of the 0.x days. But it seems to work without it, so Ill remove this.